9676 dsm component

brianpetway opened the pull request:
https://federal-spending-transparency.atlassian.net/browse/DA-9676

anjenkin reviewed (changes requested):
Looks good.  The horizontal line in the mock should be in the dsm component.  Also, we'll need to hide the demo page from upper environments by wrapping the page with the prerelease component or adding the dsm component to the quick-hits template.

brianpetway commented:
Added the component to the Quick Hits template and matched styles. Deleted the demo page that was there. Added basic unit tests.
Ready to review again.

anjenkin commented on src/components/dsm/dsm.spec.js:
```diff
@@ -0,0 +1,29 @@
+    expect(getByText('paragraph 1')).toBeTruthy();
+    expect(getByText('paragraph 2')).toBeTruthy();
+  });
+})
```
Thanks for adding the unit tests
